src/components/ContactRow.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function ContactRow(props) {
+  const { name, email, phone, onConfirmDelete } = props;
+
+  return (
+    <tr>
+      <th>{name}</th>
+      <td>{email}</td>
+      <td>{phone}</td>
+      <td>
+        <button
+          onClick={() => {
+            if (confirm(`Delete ${name}?`)) onConfirmDelete(); // eslint-disable-line no-alert, no-restricted-globals
```

Comment:
If someone knows a better way to ask for confirmation than using `confirm`, feel free to change it now.